crates/router/src/connector/iatapay/transformers.rs

```diff
@@ -8,7 +8,7 @@ use crate::{
     connector::utils::{self, PaymentsAuthorizeRequestData, RefundsRequestData, RouterData},
     core::errors,
     services,
-    types::{self, api, storage::enums},
+    types::{self, api, storage::enums, PaymentsAuthorizeData, PaymentsResponseData},
 };
```

Comment:
There is no need to import `PaymentsResponseData` and hence no need of your subsequent changes where you are updating the types path

Reply:
Done in the latest commit